javers-spring-boot-starter-mongo/src/main/java/org/javers/spring/boot/mongo/JaversMongoAutoConfiguration.java

```diff
@@ -114,9 +117,21 @@ public JaversAuditableAspect javersAuditableAspect(Javers javers, AuthorProvider
         return new JaversAuditableAspect(javers, authorProvider, commitPropertiesProvider);
     }
 
+    @Bean
+    @ConditionalOnProperty(name = "javers.auditableAspectAsyncEnabled", havingValue = "true", matchIfMissing = true)
+    public JaversAuditableAspectAsync javersAuditableAspectAsync(Javers javers, AuthorProvider authorProvider, CommitPropertiesProvider commitPropertiesProvider, @Qualifier("javersAsyncExecutor")Executor executor) {
+        return new JaversAuditableAspectAsync(javers, authorProvider, commitPropertiesProvider,executor);
+    }
+
     @Bean
     @ConditionalOnProperty(name = "javers.springDataAuditableRepositoryAspectEnabled", havingValue = "true", matchIfMissing = true)
     public JaversSpringDataAuditableRepositoryAspect javersSpringDataAuditableAspect(Javers javers, AuthorProvider authorProvider, CommitPropertiesProvider commitPropertiesProvider) {
         return new JaversSpringDataAuditableRepositoryAspect(javers, authorProvider, commitPropertiesProvider);
     }
+
+    @Bean(name = "javersAsyncExecutor")
+    @ConditionalOnMissingBean(name="javersAsyncExecutor")
+    public Executor javersAsyncExecutor(){
+        return  Executors.newFixedThreadPool(javersMongoProperties.getJaversExecutorThreadCount());
```

Comment:
NPE is throw here because `javersMongoProperties.getJaversExecutorThreadCount()` is null,
ths build is red